moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -582,6 +582,7 @@ class KinematicsBase
   KinematicsBase();
 
 protected:
+  rclcpp::Node::SharedPtr node_;
```

Comment:
When does this get initialized or used?

Reply:
The initialization happens in the kinematics_plugin, `kdl_`, `lma_`, `srv_` make use of node